lib/passport-paypal-oauth/strategy.js

```diff
@@ -44,10 +44,12 @@ function Strategy(options, verify) {
   options = options || {};
   options.authorizationURL = options.authorizationURL || 'https://www.paypal.com/webapps/auth/protocol/openidconnect/v1/authorize';
   options.tokenURL = options.tokenURL || 'https://www.paypal.com/webapps/auth/protocol/openidconnect/v1/tokenservice';
-  
+  options.userInfoURL = options.userInfoURL || 'https://www.paypal.com/webapps/auth/protocol/openidconnect/v1/userinfo?schema=openid';
```

Comment:
could you please use this approach: https://github.com/jaredhanson/passport-facebook/blob/master/lib/strategy.js#L56 ?

thanks!